### DISCUSSION TITLE
Release 1.1.0 waiting room

### DISCUSSION THREAD

**Discussion by wallhided** [kafbat/kafka-ui#563] [Q&A]
Hey!
Our team is eager to get Release 1.1.0 with bug fixes.
Especially, https://github.com/orgs/kafbat/projects/11?pane=issue&itemId=57285033

Could you please provide any estimates?

**Comment by robertcoltheart** [comment]
Yep was excited to see someone continue the work of Kafka UI, unfortunately this feels like vaporware.

**Reply by Haarolean** [reply]
We've done much work to release 1.0 in our spare time. As you can see from our initial announcement and the first major release changelog, we're not being monetarily supported anymore by any entity, yet I don't see your username among our supporters, just the demands 🙃

**Comment by Haarolean** [comment]
Hi @wallhided, you can track the progress here: https://github.com/orgs/kafbat/projects/11
We'll release it once I'm done implementing an auth page, which we considered a must-have issue for this release for various reasons. Meanwhile, you can use any desired tag (we have one for each commit pushed to the main branch).

**Comment by Haarolean** [comment]
release will be out next week (13th)
